src/main/java/br/com/sunty/models/category/CategoryDto.java

```diff
@@ -1,25 +1,26 @@
 package br.com.sunty.models.category;
 
+import java.util.List;
+
 public class CategoryDto {
     private final String name;
     private final String urlCode;
     private final String shortDescription;
     private final String guideText;
-    private final boolean isActive;
     private final Integer orderToShow;
-    private final String pathImg;
     private final String hexHtmlColor;
-
+    private final int totalCourse;
```

Comment:
Aqui acho que seria no plural. E no getter dessa classe também.

```suggestion
    private final int totalCourses;
```